### PR TITLE
[TASK]: Recording sorted in Descending order

### PR DESCRIPTION
**Pull request by nayanasQA**
Listing the recordings in Ascending order.

### CODE REVIEW

**Review by meyyappanv — APPROVED** [review]

File: view.php [hunk 1]
```diff
@@ -184,12 +184,14 @@
 $table->data[] = array($strautorec, get_string('auto_rec_' . $zoom->auto_recording, 'mod_zoom'));
 
 $records = get_zoom_meeting_recordings($zoom->meeting_id);
```
Comment (meyyappanv) [inline view.php:186]: @nayanalinkstreet why not implement order by in the query in the method get_zoom_meeting_recordings itself? Ne need to retrieve the recording one more time.
Also try avoiding the query here in this file directly.

File: view.php [hunk 2]
```diff
@@ -184,12 +184,11 @@
...
-
 if (!empty($records)) {
     if($zoom->enable_stream_url == 1 && $zoom->enable_download_url == 1){
         //$table->data[] = array(get_string('view_recording','zoom'));
         $display = '';
-        foreach ($records as $key => $value) {
+        foreach ($records as $value) {
```
Comment (meyyappanv) [inline view.php:191]: @nayanalinkstreet there shouldn't be any change this file at all.

File: locallib.php [hunk 3]
```diff
@@ -663,5 +663,6 @@ function get_zoom_meeting_recordings($meeting_id) {
     return $DB->get_records_sql("SELECT rec.play_url,rec.download_url,
         rec.status,rec.start_time
         from mdl_zoom_recordings as rec
-        where rec.meeting_id = {$meeting_id}");
+        where rec.meeting_id = {$meeting_id}
+        ORDER BY start_time ASC");
```
Comment (meyyappanv) [inline locallib.php]: @nayanalinkstreet I believe this will be rec.start_time ?

Comment (Naveen1895) [inline locallib.php]: @nayanalinkstreet Can we make this descending? So that the latest recordings come on top?

**Review by Naveen1895 — APPROVED** [review]

**Review by meyyappanv — APPROVED** [review]

File: view.php (hunk 2 above)
Comment (nayanasQA) [reply view.php:191]: @meyyappanv Yes, there is no change for this file.

**Pull request merged**